ansible/roles/ant/files/ant.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+ANT_HOME=/opt/ant/apache-ant-1.10.10
```

Comment:
Please use a template and use the existing `ant_version` variable

Reply:
made changes